### DISCUSSION TITLE
Driver SSD1963 parallel STM32

### DISCUSSION THREAD

**Discussion by domynew** [Bodmer/TFT_eSPI#2364] [Compatible support libraries]
Sto collegando un display 7" SSD1963 parallel 8 bit, la piattaforma usata è NUCLEO STM32F767ZI

Nessun setup SSD1963 lavora con la piattaforma usata NUCLEO STM32F767ZI

Con la stessa piattaforma NUCLEO STM32F767ZI ho testato altri display con driver diversi e lavorano benissimo

Esiste un rimedio? 

Grazie per il supporto.

![connettore](https://user-images.githubusercontent.com/123870977/215325857-4999a23f-c154-499e-b177-2149bee80304.jpg)

![Immagine 2023-01-29 132952](https://user-images.githubusercontent.com/123870977/215326211-c2a9bd92-1c9f-4a9e-96f7-654bff6e1373.jpg)

**Comment by Bodmer** [comment]
Post a link to the display vendors web page so I can review the technical details.

**Reply by domynew** [reply]
https://eckstein-shop.de/72C022800x480TFTLCDDisplaymitTouchscreenSSD1963MCUArduinoKompatibel

**Comment by Bodmer** [comment]
That board will not plug straight into the NUCLEO board so I assume you have wired it up somehow.

List all connections between the NUCLEO board and the display board.

**Reply by domynew** [reply]
DISPLAY to NUCLEO (8 bit)

RD to A0
WR to A1
RS to A2
CS to A3
REST to A4

D0 to D8
D1 to D9
D2 to D2
D3 to D3
D4 to D4
D5 to D5
D6 to D6
D7 to D7

LED_A to +3.3V
3.3V to +3.3V
5V to +5V
GND to GND
